Source/UserManagement/Events/DataCollector/PhoneNumberAddedToDataCollector.cs

```diff
@@ -0,0 +1,20 @@
+using doLittle.Events;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Events.DataCollector
+{
+    public class PhoneNumberAddedToDataCollector : IEvent
+    {
+        // QUESTION: Needed? Is this EventSourceId? If so, then all events should have this? public Guid Id { get; set; }
```

Comment:
I can't see how this is needed, events should only provide information relevant to the event. Comments, @einari?

Reply:
@einari for further information, the property in question is a
public Guid id

Reply:
@karolikl May have been something that I forgot to remove after a copy-paste or something also